layers/+email/mu4e/packages.el

```diff
@@ -46,6 +47,10 @@
         (add-hook 'mu4e-compose-pre-hook 'mu4e/set-account)
         (add-hook 'message-sent-hook 'mu4e/mail-account-reset)))))
 
+(defun mu4e/init-mu4e-alert ()
+  (use-package mu4e-alert
+    :commands (mu4e-alert-enable-notifications)))
+
```

Comment:
Yes, you should go with `:defer t` and not use `:commands` as the functions you want to use can be autoloaded.

> I have a question about the names of the layer variables. The mu4e- prefix is used by mu4e, as a result naming variables with that prefix might at some point conflict with mu4e. Is there any other way to name those variables? Maybe spacemacs-mu4e-xxx? (it gets too long though)

That's a very valid point and even I'm confused at this stage. I know there can be a conflict. We should definitely address this at this stage. @syl20bnr @TheBB @StreakyCobra Any suggestions on how to go about defining layer variables and avoid namespace pollution? 

You should have enough ground to write better layers after going through this https://github.com/syl20bnr/spacemacs/blob/develop/doc/LAYERS.org